CHANGELOG.md

```diff
@@ -499,7 +499,15 @@ Other minor additions
   ≢-sym : Symmetric _≢_
   ```
 
+* Defined some new utilities of unary list relations:
```

Comment:
Could you split this up like the rest of the CHANGELOG? i.e. put `uncons` under the entry for `Data.List.Relation.Unary.All` and the rest under `Data.List.Relation.Unary.Any.Properties`.

Reply:
Yes, done